client/src/containers/LocationForm.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Redirect } from 'react-router-dom';
+import Select from 'react-select';
+import 'react-select/dist/react-select.css';
+import * as actions from '../actions';
+import { locationdata } from '../location_data';
+
+class LocationForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { selectedOption: '', redirect: false };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(selectedOption) {
+    this.setState({ selectedOption });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    this.props.chooseLocation(this.state.selectedOption.value);
+    this.setState({ redirect: true });
+  }
+  render() {
+    if (this.state.redirect) {
+      return <Redirect to="/" />;
```

Comment:
@hoslack The next page after location will be for uploading files so maybe you can already redirect to '/upload'